apps/web/src/components/StudentSettings/ProfileForm.tsx

```diff
@@ -28,6 +26,9 @@ import { useInvalidateQuery } from "@litespace/headless/query";
 import { QueryKey } from "@litespace/headless/constants";
 import { useUpdateUser } from "@litespace/headless/user";
 import { useMediaQuery } from "@litespace/headless/mediaQuery";
+import NotificationSettings from "@/components/Common/NotificationSettings";
```

Comment:
The nofication settings can be moved into `components/SharedSettings/` as well as the topic selection related components.